integration_tests/src/main/python/cast_test.py

```diff
@@ -151,6 +151,7 @@ def test_cast_string_date_non_ansi():
                                       StringGen('[0-9]{1,4}-[0-3][0-9]-[0-5][0-9][ |T][0-3][0-9]:[0-6][0-9]:[0-6][0-9]'),
```

Comment:
Not sure why ```test_cast_string_date_valid_format```(In L65 above) and ```test_cast_string_date_non_ansi```(In L144 above) are not touched. No requirement for Spark.